# A n-dimensional vector state
float32[] data